SallyLibrary.App.Tests.Unit/Services/Foundations/BookServiceTests.Logic.RemoveById.cs

```diff
@@ -0,0 +1,43 @@
+// ---------------------------------------------------------------
+// Copyright (c) Coalition of the Good-Hearted Engineers
+// FREE TO USE AS LONG AS SOFTWARE FUNDS ARE DONATED TO THE POOR
+// ---------------------------------------------------------------
+
+using System;
+using FluentAssertions;
+using Force.DeepCloner;
+using Moq;
+using SallyLibrary.App.Brokers.Storages;
+using SallyLibrary.App.Models.Books;
+using SallyLibrary.App.Services.Foundations.Books;
+using Xunit;
+
+namespace SallyLibrary.App.Tests.Unit.Services.Foundations
+{
+    public partial class BookServiceTests
+    {
+        [Fact]
+        public void ShouldRemoveBookById()
+        {
+            //given
+            Book randomBook = new Book();
+            Guid inputBookId = randomBook.Id;
+
```

Comment:
Single lines of code doesn't need an empty line between them.
Please refer to our C# Coding Standard
https://github.com/hassanhabib/CSharpCodingStandard/blob/master/Variables.md#dont-9